client/src/styles/settingscontainer.scss

```diff
@@ -0,0 +1,30 @@
+.settings-justify-between {
+    display: flex;
+    justify-content: space-between;
+}
+.settings-justify-center {
+    display: flex;
+    justify-content: center;
+}
+.column {
```

Comment:
these will now over-write same classes in general.scss, which is not good. We should think about the player settings layout a bit